app/controllers/projects_controller.rb

```diff
@@ -18,7 +18,10 @@ def get_projects
   # GET /projects/1
   # GET /projects/1.json
   def show
-    @project.increase_views(current_user)
+    if current_visit && !Ahoy::Event.find_by(visit_id: current_visit.id)
+      ahoy.track("Visited project " + @project.id.to_s)
+      @project.increase_views(current_user)
```

Comment:
This is old logic, we can fetch this from db and set counter_cache for faster access